### PR TITLE
I18n fixes

### PR DESCRIPTION
**Pull request by aronasorman**
## Summary

Some i18n fixes to get ready for the 0.16 po file release.

## Reviewer guidance

Review by commit. Changes will make more sense.

### CODE REVIEW

**Comment by aronasorman** [comment]
Tests are now passing.

File: kalite/i18n/management/commands/makemessages.py [hunk 1]
```diff
@@ -30,32 +33,42 @@ def handle_noargs(self, *args, **options):
...
+            for entry in inline_help_poentries:
+                po.append(entry)
+            po.save(pofile_path)
 
+    def extract_inline_help_strings(self, inline_help_path=None):
+        '''
+        Extract the strings from the inline help narratives yml files. Returns an
+        iterator containing the po file entries. Optional inline_help_parameter
+        specifies where the inline help narratives path is. Else, it defaults
+        to settings.CONTENT_DATA_PATH + "narratives.yml"
```
Comment (MCGallaspy) [inline kalite/i18n/management/commands/makemessages.py]: >Optional inline_help_parameter
specifies where the inline help narratives path is. Else, it defaults
to settings.CONTENT_DATA_PATH + "narratives.yml"

It appears to be an unused parameter.

**Comment by MCGallaspy** [comment]
Looking good, great commit history! Just one comment above on a docstring/unused parameter.

**Comment by aronasorman** [comment]
Addressed comment. Tests pass.

**Pull request merged**